libensemble/gen_classes/gpCAM.py

```diff
@@ -74,7 +74,7 @@ def ask_numpy(self, n_trials: int) -> npt.NDArray:
                 input_set=np.column_stack((self.lb, self.ub)),
                 n=n_trials,
                 pop_size=n_trials,
-                acquisition_function="total correlation",
```

Comment:
This should prob be passed as option, with sensible default.

Reply:
But also there is "MINIMIZE", "MAXIMIZE" keywords. If we added 'EXPLORE', that would probably be "total correlation".